README.md

```diff
@@ -1,8 +1,8 @@
 # VMware Carbon Black Cloud Python SDK
 
-**Latest Version:** 1.4.3
+**Latest Version:** 1.5.0
 <br>
-**Release Date:** June 26, 2023
+**Release Date:** TBD
```

Comment:
We should aim for today so when you need too update the date for today

Reply:
That's in Step 1 of the "Releasing the SDK" process.